packages/playground/blueprints/src/lib/utils/logger.ts

```diff
@@ -0,0 +1,140 @@
+import { PHPRequestEndEvent } from "@php-wasm/universal/src/lib/universal-php";
+import { UniversalPHP } from "@php-wasm/universal/src/lib/universal-php";
+/**
+ * Log severity levels.
+ */
+export enum LogSeverity {
```

Comment:
Let's avoid TypeScript enums, they have weird properties and can lead to confusing errors – I have a bunch of notes that aren't distilled enough for this comment, but I'm happy to discuss the next time we chat. One alternative would be to use a string union, like `"DEBUG" | "INFO"` etc.

Reply:
Ok, I removed the enum. I like using them, but string unions also work here.